### ISSUE TITLE
Math for K to F

### ISSUE THREAD

**Issue by Drag91mustang** [abhishtagatya/piWeatherman#4]
I was getting a really off number for this. I check and fixed the code here.  
 k_to_f = int(weather_now['main']['temp'] - 273.15) * (1.8) + (32)

**Comment by abhishtagatya** [comment]
right, sorry for that i guess i didn't really check, please submit it as a pull request.
